ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+            contentView.topAnchor.constraint(equalTo: bottomBarView.topAnchor, constant: Constants.BottomBar.heroStackTopBottomMargin),
+            contentView.bottomAnchor.constraint(equalTo: bottomBarView.bottomAnchor, constant: -Constants.BottomBar.heroStackTopBottomMargin)
+        ])
+
+        return bottomBarView
+    }
+
+    private func makeBottomSheetContent(headerView: UIView, expandedContentView: UIView) -> UIView {
+        let view = UIView()
+        let separator = Separator()
+        separator.translatesAutoresizingMaskIntoConstraints = false
+        headerView.translatesAutoresizingMaskIntoConstraints = false
+        expandedContentView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(headerView)
+        view.addSubview(expandedContentView)
+        view.addSubview(separator)
+        NSLayoutConstraint.activate([
+            headerView.topAnchor.constraint(equalTo: view.topAnchor),
+            headerView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            headerView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            expandedContentView.topAnchor.constraint(equalTo: headerView.bottomAnchor),
+            expandedContentView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            expandedContentView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            expandedContentView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            separator.topAnchor.constraint(equalTo: expandedContentView.topAnchor),
+            separator.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            separator.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        return view
+    }
+
+    private func updateExpandability() {
+        if isInSheetMode {
+            bottomSheetController?.collapsedContentHeight = bottomSheetHeroStackHeight
+            bottomSheetController?.isExpandable = isExpandable
+            bottomSheetHeroStackTopConstraint?.constant = bottomSheetHeroStackTopMargin
+        } else {
+            moreButtonView.isHidden = !isExpandable
+        }
+    }
+
+    private lazy var moreButtonView: UIView = {
+        let moreButtonItem = TabBarItem(title: Constants.BottomBar.moreButtonTitle, image: Constants.BottomBar.moreButtonIcon ?? UIImage())
+        let moreButtonView = TabBarItemView(item: moreButtonItem, showsTitle: true)
+        moreButtonView.alwaysShowTitleBelowImage = true
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleMoreButtonTap(_:)))
+        moreButtonView.addGestureRecognizer(tapGesture)
+
+        NSLayoutConstraint.activate([
+            moreButtonView.widthAnchor.constraint(equalToConstant: Constants.heroButtonWidth),
+            moreButtonView.heightAnchor.constraint(equalToConstant: Constants.heroButtonHeight)
+        ])
+
+        return moreButtonView
+    }()
```

Comment:
do you actually need the parenthesis here? I would expect not

Reply:
You do, otherwise swift thinks you are assigning a closure to the property.